prisma/schema.prisma

```diff
@@ -327,3 +327,12 @@ model Webhook {
   eventTriggers   WebhookTriggerEvents[]
   user            User                   @relation(fields: [userId], references: [id])
 }
+
+model UserSettings {
+  id        Int      @id @default(autoincrement())
+  userId    Int
+  key       String
+  value     String
```

Comment:
Why not use a `jsonb` column instead?

Reply:
I've been thinking the same thing. We these user settings could be namespaced in the metadata field:

```
metadata: {
  "has_web3_enabled": true,
  // or...
  "web3": { "enabledAt": Datetime }
  // or something similar...
}
```